lib/Minz/Request.php

```diff
@@ -227,6 +225,42 @@ public static function getBaseUrl() {
 		return filter_var($url, FILTER_SANITIZE_URL);
 	}
 
+	/**
+	 * Test if a given server address is publicly accessible.
+	 *
+	 * Note: for the moment it tests only if address is corresponding to a
+	 * localhost address.
+	 *
+	 * @param $address the address to test, can be an IP or a URL.
+	 * @return true if server is accessible, false otherwise.
+	 * @todo improve test with a more valid technique (e.g. test with an external server?)
+	 */
+	public static function serverIsPublic($address) {
+		if (strlen($address) < strlen('http://a.bc')) {
```

Comment:
We're sure it will always includes a protocol at this point?

Reply:
If it does not, then it is not a valid public address